README.md

```diff
@@ -610,6 +610,19 @@ You can disable this integration by doing
 let g:test#no_alternate = 1
 ```
 
+### Custom alternate file
+
+If you are using a different library for jumping between implementation and test file
+you can define a custom function that returns the test filename.
+
+```vim
+function! CustomAlternateFile(cmd)
+  return "test_file_spec.rb"
+endfunction
+
+let g:test#custom_alternate_file = function('echo')
```

Comment:
Shouldn't that be `CustomAlternateFile` instead of `echo`?

Reply:
Good question, @gmartsenkov what are your thoughts? Perhaps we need some automated tests around this